apis/appmesh/v1beta2/types.go

```diff
@@ -226,3 +83,11 @@ type MeshReference struct {
 	// UID is the UID of Mesh CR
 	UID types.UID `json:"uid"`
 }
+
+// VirtualGatewayReference holds a reference to VirtualGateway.appmesh.k8s.aws
+type VirtualGatewayReference struct {
+	// Name is the name of Mesh CR
+	Name string `json:"name"`
+	// UID is the UID of Mesh CR
+	UID types.UID `json:"uid"`
```

Comment:
Shouldn't specifying `Namespace` here be sufficient as VirtualGateway is a Namespace-scoped resource unlike Mesh?

Reply:
Yes thats right. This has to be namespace-scoped. Will change it.